envs/onnx-env.yaml

```diff
@@ -11,9 +11,14 @@ packages:
   - feedstock : nlohmann_json
   - feedstock : https://github.com/conda-forge/fire-feedstock
     git_tag : a691d408ec35d71fdf9aecbed6efb408affb2b80
+{% if build_type == 'cuda' %}
+  - feedstock : cudatoolkit    #[cudatoolkit == "11.2"]
+  - feedstock : libc_wrappers     #[x86_64 and (cudatoolkit == '11.0' or cudatoolkit == '11.2' or (py_ver < 3.8 and cudatoolkit == "10.2" ))]
+  - feedstock : cudnn
+  - feedstock : nccl
+{% endif %}
   - feedstock : onnxruntime
   - feedstock : onnxconverter-common
-  - feedstock : keras2onnx
```

Comment:
@jayfurmanek - Should we keep keras2onnx in 1.3? keras2onnx is used onnxmltools. 
For main branch (1.4 release), I removed keras2onnx from this file as well as updated onnxmltools to the newer version which doesn't depend on keras2onnx. But for 1.3, I think we can keep keras2onnx and onnxmltools as they are tagged.

Reply:
Currently the PR check is failing because kers2onnx is removed from the onnx-env.yaml file but it is used by onnxmltools tagged in 1.3. To fix the PR check, either I have to add keras2onnx back in the onnx-env.yaml or I've to update onnxmltools to not depend on it which means re-tagging of onnxmltools from open-ce-v1.3.

Reply:
yes, correct.